scripts/npcs/village/recruitmaurice.lua

```diff
@@ -35,22 +35,22 @@ local function recruit_talk(npc, ch)
         local res = ask(choices)
 
         if res == 1 then
-            say("Oh, I just don't want to spent the rest of my life with "
+            say("Oh, I just don't want to spent the rest of my life doing "
                 .. "farm work. You know, I've got so many brothers and "
-                .. "sisters, I didn't like the thought to drudge on a dirty "
+                .. "sisters, I didn't like the thought of drudging on a dirty "
                 .. "farm all my life for my oldest brothers benefit. "
                 .. "So I ran away when I was twelve.")
-            say("I tramped around the area some while, finding some work to "
+            say("I tramped around the area for some while, finding some work to "
```

Comment:
Probably nicer "for a while", especially since "some" is used again later in the sentence.

Reply:
I'll also get that.